faculty_cli/cli.py

```diff
@@ -710,24 +710,12 @@ def instance_types(verbose):
             click.echo(type_.name)
 
 
-@server.command(name="add-to-ssh-agent")
+@server.command(name="ssh-details")
 @click.argument("project")
 @click.argument("server")
-def ssh(project, server):
-    """
-    Add SSH private key for a Faculty server into the SSH authentication agent
-    and print the username, hostname and port.
-
-    After running this command, SSH into the server using
-    `ssh <username>@<hostname> -p <port>`.
-    """
+def ssh_details(project, server):
+    """Echo the username, hostname and SSH port for a Faculty server. """
```

Comment:
Trailing space inside the docstring.

I think expanding the docstring with what you had before would also be useful:

```
After running this command, SSH into the server using
`ssh <username>@<hostname> -p <port>`.
```

... with a sentence about needing to add the key to `~/.ssh/authorized_keys`?

Reply:
`~/.ssh/authorized_keys` on the platform, right?

Reply:
I've updated the docstring. Please check your okay with the wording (I still get confused how to refer to the Platform)